src/ra_fwd.h

```diff
@@ -5,25 +5,36 @@
 // Forward declaring namespace std caused problems
 #include <xstring>
 
-
-namespace ra {
-
 #ifndef _TCHAR_DEFINED
 #if _MBCS
-using TCHAR = char;
+using TCHAR  = char;
+using PTCHAR = char*;
+using TBYTE  = unsigned char;
+using PTBYTE = unsigned char*;
 #elif _UNICODE
-using TCHAR = wchar_t;
+using TCHAR  = wchar_t;
+using PTCHAR = wchar_t*;
+using TBYTE  = wchar_t;
+using PTBYTE = wchar_t*;
 #else 
 #error Unknown character set detected, only MultiByte and Unicode are supported!
 #endif // _MBCS
 #define _TCHAR_DEFINED
 #endif // !_TCHAR_DEFINED
 
+#ifndef _WINDEF_
+using DWORD  = unsigned long;
+using LPVOID = void*;
+#endif // !_WINDEF_
 
+#ifndef _WINNT_
+using HANDLE = void*;
+#endif // !_WINNT_
+
```

Comment:
Depending on where this file is included will determine if they need to be aliased or not to prevent a "redefinition".